src/test/test-cgroup.c

```diff
@@ -159,6 +159,8 @@ TEST(id) {
 
         if (ERRNO_IS_NEG_PRIVILEGE(fd2))
                 log_notice("Skipping open-by-cgroup-id test because lacking privs.");
+        else if (fd2 == -ENOSYS)
```

Comment:
I think `ERRNO_IS_NEG_NOT_SUPPORTED(fd2)` is better.

Reply:
Thanks, done.